src/networking/postApiRequest.ts

```diff
@@ -17,9 +17,9 @@ const postApiRequest = async (request: MCMCMonitorRequest): Promise<MCMCMonitorR
         }
     )
     const response = await rr.json()
-    if (!isMCMCMonitorResponse) {
+    if (!isMCMCMonitorResponse(response)) {
         console.warn(response)
-        throw Error('Unexpected api response')
+        throw TypeError('Unexpected api response')
```

Comment:
Not really quite right, but I'd like to move toward using more precise error types when possible. Need to look into whether we can do custom types as easily in JS as we can in Python.